ephios/core/views/accounts.py

```diff
@@ -134,7 +134,8 @@ def post(self, request, *args, **kwargs):
                     name=self.object.get_full_name(), user=self.object
                 ),
             )
-            ProfileUpdateNotification.send(userprofile)
+            if request.user != userprofile:
+                ProfileUpdateNotification.send(userprofile)
```

Comment:
might want to inform the user about password changes, even if it was done through their account

Reply:
password changes cant't happen via this view